scipy/sparse/linalg/_eigen/_svds.py

```diff
@@ -6,7 +7,8 @@
 from scipy._lib._util import check_random_state
 from scipy.sparse.linalg._interface import LinearOperator, aslinearoperator
 from scipy.sparse.linalg._eigen.lobpcg import lobpcg  # type: ignore[no-redef]
-from scipy.sparse.linalg._svdp import _svdp
+if os.environ.get("USE_PROPACK"):
+    from scipy.sparse.linalg._svdp import _svdp
```

Comment:
Can you add a check under `elif solver == 'propack':`, something like:
```python
HAS_PROPACK = False

....

if not HAS_PROPACK:
    raise ValueError("`solver='propack'` is opt-in due to potential issues on Windows, "
                      "it can be enabled by setting the `USE_PROPACK` environment "
                      "variable before importing scipy")

Reply:
This looks good to me.